framework/src/org/checkerframework/common/value/ValueAnnotatedTypeFactory.java

```diff
@@ -1242,15 +1242,19 @@ public AnnotationMirror createIntRangeAnnotation(long from, long to) {
     }
 
     /**
-     * Create an {@code @IntRange} annotation from the range. May return BOTTOMVAL or UNKNOWNVAL.
+     * Create an {@code @IntRange} or {@code @IntVal} annotation from the range. May return
+     * BOTTOMVAL or UNKNOWNVAL.
      */
     public AnnotationMirror createIntRangeAnnotation(Range range) {
```

Comment:
Should this be combined with ValueAnnotatedTypeFactory#createIntValAnnotation to create one unified createIntAnnotation function, which can take either a range or a list of values (and then converts one to the other before calling one unified method)?